dev-python/pycollada/pycollada-0.6-r1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python{2_7,3_{5,6,7}} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library for reading and writing COLLADA documents"
+HOMEPAGE="https://pycollada.readthedocs.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	>=dev-python/python-dateutil-2.2[${PYTHON_USEDEP}]
+"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+"
```

Comment:
You sure that there are no test dependencies? Not even RDEPEND required for tests?

Reply:
Tests need `python-dateutil`, which was in `DEPEND` and was removed upon your request (https://github.com/gentoo/gentoo/pull/12253#pullrequestreview-253087458 second resolved item).

Isn't it guaranteed that all `RDEPEND` dependencies are installed before package gets emerged? If that's the case, we wouldn't need it in `DEPEND` for the tests, as it gets unconditionally pulled in anyway in `RDEPEND`. 

Would a statement like `test? ( ${RDEPEND} )` be valid?

Reply:
No, RDEPEND are not guaranteed during binary package builds. And yes, the last construct is what I had in mind.

Reply:
Oh I see. So for binary packages, at build time only DEPEND is checked, and when they get emerged, only RDEPEND is checked? Well that makes sense. Thanks for clarifying this!
I'm gonna change it.